service/policy/objects.proto

```diff
@@ -279,15 +339,13 @@ message KasPublicKey {
   string kid = 2;
 
   // A known algorithm type with any additional parameters encoded.
-  // To start, these may be `rsa:2048` for encrypting ZTDF files and 
+  // To start, these may be `rsa:2048` for encrypting ZTDF files and
   // `ec:secp256r1` for nanoTDF, but more formats may be added as needed.
   KasPublicKeyAlgEnum alg = 3;
 }
 
 // A list of known KAS public keys
-message KasPublicKeySet {
-  repeated KasPublicKey keys = 1;
-}
+message KasPublicKeySet { repeated KasPublicKey keys = 1; }
```

Comment:
Looking at this makes me wonder - do we want to get rid of the concept of a public key set alongside the deprecation of grants? If so, can we document that?

Reply:
Yeah that should go away. I believe I only leveraged `KasPublicKey`. I will go through and add deprecation comments to that and kas grants.